source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -51,12 +51,65 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
 
       $ sudo systemctl restart wazuh-manager
 
-Visualize the alerts
---------------------
+Visualizing the alerts
+======================
 
-You can visualize the alert data in the Wazuh dashboard. To do this, go to the Vulnerability Detection module, select Events, and click on any vulnerability to expand the document.
+To access information about vulnerabilities detected in your system, follow these steps:
+
+1. **Navigate to the Thread Intelligence Section**: In the Wazuh dashboard, locate the "Thread Intelligence" section. This is where you'll find insights into potential threats and vulnerabilities.
+
+2. **Find the Vulnerability Detection Module**: Within the Thread Intelligence section, look for the "Vulnerability Detection" module. Click on it to access detailed information about vulnerabilities across all endpoints.
+
+.. thumbnail:: /images/poc/vulnerabilities-section.png
+   :title: Detected vulnerability detection section.
+   :align: center
+   :width: 80%
+
+Dashboard
+---------
+
+This dashboard offers valuable insights into your system's vulnerabilities. It includes:
+
+- **Summary of Top 5 Vulnerabilities**: Quickly identify the most critical vulnerabilities that require immediate attention.
+  
+- **Distribution of Vulnerabilities by Severity Levels**: Understand the severity distribution of vulnerabilities, helping you prioritize your response effectively.
+
+- **Total Number of Vulnerabilities**: Get an overview of the total number of vulnerabilities grouped by their severity levels.
```

Comment:
Order them as they appear on the dashboard
- Distribution of Vulnerabilities by Severity Levels
- Summary of Top 5 Vulnerabilities
- Most common vulnerabilities

"Total Number of Vulnerabilities" is not present on this section

Reply:
Done!